VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/services/video-call.service.ts

```diff
@@ -183,7 +184,10 @@ export class VideoCallService {
         this.onErrorSubject.next(new CallError(error));
     }
 
-    private handleDisconnect(reason: string) {
+    // Handles server issued disconections - NOT CLIENT
+    // https://docs.pexip.com/api_client/api_pexrtc.htm#disconnect
+    private handleAdminDisconnect(reason: string) {
```

Comment:
Maybe call this something else since admin disconnected is just an example. https://docs.pexip.com/api_client/api_pexrtc.htm#onDisconnect

```suggestion
    // https://docs.pexip.com/api_client/api_pexrtc.htm#onDisconnect
    private handleServerDisconnect(reason: string) {
```